client/src/components/new-request/main-container.vue

```diff
@@ -1,14 +1,18 @@
 <template>
-  <v-container class="main-container-style px-9 py-5 copy-normal">
+  <v-container class="white rounded copy-normal">
```

Comment:
Changes in this component were necessary for the proper layout of the Existing Request Display page.
However, this may have broken other pages that use Main Container. Can someone help me test and update those please?

Reply:
oo yea, this will be far reaching. I can certainly help, but i've got a mountain of work in this current MRAS ticket 😅 

To be honest, i think the only major thing to consider is the max-width this sets throughout the app. Is that enforced some other way?

Reply:
It looks OK to me atm, but I've only tested:
- New Name Search page
- Existing Name Request page
- Existing Request Display page

I'm assuming the padding will be off on the other pages, but responsive-wise it seems OK.

If anyone knows how to get to the other pages that use Main Container, please tell me!